vault/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: Vault Secret Management
 name: vault
-version: 1.1.3
+version: 1.1.4
```

Comment:
Is a patch version update appropriate here?

Reply:
i think it could probably be a minor instead of a patch. i'll adjust that now before merging